.ci/travis/travis.sh

```diff
@@ -450,7 +450,7 @@ no-exception-test-alot-of-project1)
 
 no-error-pmd)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
-                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
```

Comment:
please remove this change (or move it to separate commit of the same PR), it is better to not do unrelated changes in same commit.